src/cli/cli_srp_client.cpp

```diff
@@ -209,6 +344,23 @@ template <> otError SrpClient::Process<Cmd("host")>(Arg aArgs[])
             IgnoreError(otSrpClientSetHostAddresses(GetInstancePtr(), hostAddressArray, numAddresses));
         }
     }
+    /**
+     * @cli srp client host remove
+     * @code
+     * srp client host remove 1
+     * Done
+     * @endcode
+     * @cparam srp client host remove [@ca{removekeylease}] [@ca{sendunregtoserver}]
+     *  * The parameter `removekeylease` is an optional boolean value that indicates
+     *    whether the host key lease should also be removed (default is `false`).
+     *  * The parameter `sendunregtoserver` is an optional boolean value that indicates
+     *    whether the client host should send an "update" message to the server
+     *    even when the client host information has not yet been registered with the
+     *    server (default is `false`). This parameter can be specified only if the
+     *    `removekeylease` parameter is specified first in the command.
+     * @par api_copy
+     * #otSrpClientRemoveHostAndServices
```

Comment:
So for some of these commands, copying the API description doesn't work so well, since it's worded in terms of functions and using the API (rather than the CLI).

I think for some of these, we either need to 1) not copy the API description and just add any relevant info directly here or 2) rewrite the API description so that it works for both the API and command.  For some,  option 2 won't work so well.  Like for this one, you added relevant info for the command parameters but this duplicates what's in the description.

Ideally you should be able to read just the content on the CLI commands page and it should make sense in the context of that command without having to go to the API docs.

Can you check throughout these and make any necessary adjustments?

Reply:
yeah, i thought of the same thing ....will definitely go through it all with this in mind...thanks